kuma/wiki/templates/wiki/document.html

```diff
@@ -399,6 +416,15 @@ <h2 class="offscreen">{{ _('Document Tags and Contributors') }}</h2>
     <menuitem data-action="$edit" label="{{_('Edit page')}}"></menuitem>
     <menuitem data-action="$history" label="{{_('View page history')}}"></menuitem>
   </menu>
+
+  <!-- embed scrollback widget based on the tags on the page -->
+  {% set tags = document.tags.all() %}
+  {% for tag in tags %}
+    {% if settings.SCROLLBACK_ROOM_MAP[tag.name] and waffle.flag('scrollback_widget') %}
+      <button class = "scrollback-button" onclick='addScrollback("{{settings.SCROLLBACK_ROOM_MAP[tag.name]}}")'> Chat about {{tag.name}} </button>
+    {% endif %}
+  {% endfor %}
+
```

Comment:
I don't understand this block, why do we have to go through the list of document tags if the setting for this only contains one item?

The call to `settings.SCROLLBACK_ROOM_MAP[tag.name]` will also raise a KeyError as soon as one of the tags don't match.